org.flatpak.demo.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "org.flatpak.demo",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
+    "command": "flatpak-demo",
+    "rename-icon": "flatpak",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "cflags": "-O2 -g",
```

Comment:
Probably remove these and just use `"config-opts": ["-DCMAKE_BUILD_TYPE=RelWithDebInfo"]`